packages/core/src/api/FirmwareUpdateV2.ts

```
@@ -103,17 +123,50 @@
 
     // check device goto bootloader mode
     let isFirstCheck = true;
+    let checkCount = 0;
+    // eslint-disable-next-line prefer-const
+    let timeoutTimer: ReturnType<typeof setTimeout> | undefined;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'timeoutTimer' is only assigned here.


(lint/style/useConst)